client/asset/eth/multirpc_live_test.go

```diff
@@ -102,6 +105,33 @@ func testEndpoint(endpoints []string, syncBlocks uint64, tFunc func(context.Cont
 	return nil
 }
 
+func TestMain(m *testing.M) {
+	var cancel context.CancelFunc
+	ctx, cancel = context.WithCancel(context.Background())
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+	go func() {
+		select {
+		case <-c:
+			cancel()
+		case <-ctx.Done():
+		}
+	}()
+	exit := func(exitCode int) {
+		signal.Stop(c)
+		cancel()
+		os.Exit(exitCode)
+	}
+	time.Sleep(time.Second)
+	err := mine(ctx)
+	if err != nil {
+		fmt.Println(err)
+		exit(1)
+	}
+	time.Sleep(time.Second)
+	exit(m.Run())
+}
```

Comment:
Why need all this?

Reply:
I was mining two blocks before tests when I took out the Simnet check for old headers, but then put the Simnet check back. So.. can revert.